workflow/apm/database.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Databases
+sidebarTitle: Databases
+---
+
+The Databases page displays all of your database data that flows through your instrumented APMs. This page is useful for monitoring high-level database interactions and how they can directly impact the performance of other key applications within your software architecture. 
```

Comment:
Still too much use of the word database. General rule of thumb is if you have to use the same word twice in a sentence it's not being described clearly enough. A way I would reword this is "The Databases page displays all queries and runtime metrics related to requests your application makes."